### PR TITLE
Move all font loading to client-side.

### PR DESCRIPTION
**Pull request by jpetto**
- Update .gitignore to only ignore root /static folder.

Tested in IE 7 - 10 and A-grade browsers.

### CODE REVIEW

**Comment by jpetto** [comment]
@schalkneethling - Would you happen to have time for a code review, good sir?

**Comment by schalkneethling** [comment]
@jpetto Taking a look, initial fly by looks all good.

File: jinja2/includes/config.html [hunk 1]
```diff
@@ -9,39 +12,37 @@
...
-            wiki: {
-                autosuggestTitleUrl: '{{ url('wiki.autosuggest_documents') }}'
+        // Site configuration
+        win.mdn.ckeditor = {};
+        win.mdn.features = {};
+        win.mdn.staticPath = '{{ STATIC_URL }}';
+        win.mdn.optimizely = win['optimizely'] || [];
+        win.mdn.wiki = {
+            autosuggestTitleUrl: '{{ url('wiki.autosuggest_documents') }}'
+        }
```
Comment (schalkneethling) [inline jinja2/includes/config.html]: Missing semicolon

File: kuma/static/js/font-check.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            'loaded': false,
+            'varient' : [
+                {'weight' : 'normal'},
+                {'weight' : 'bold'},
+                {'style' : 'italic'}
+            ]
+        }
+    ];
+
+    for (var i = 0; i < fonts.length; i++) {
```
Comment (schalkneethling) [inline kuma/static/js/font-check.js]: Not a lot of iterations here probably but, would it be worth to cache the length of the fonts array?

```
for (var i = 0, l = fonts.length; i < l; i++) {
```

File: kuma/static/js/fonts.js [hunk 3]
```diff
@@ -1,122 +1,53 @@
...
+    // loads all varients of a font
+    // accepts a font object as defined in font-check.js
+    function loadFont(fontObj) {
+        // array to hold promises for each varient
+        var varientPromises = [];
+        // holds reference to an instance of FontFaceObserver in loop below
+        var observer;
+
+        // loop over each varient for the given font
+        for (var i = 0; i < fontObj.varient.length; i++) {
```
Comment (schalkneethling) [inline kuma/static/js/fonts.js]: Again here, might be worth caching the length of the varient array?

File: kuma/static/js/fonts.js [hunk 4]
```diff
@@ -1,122 +1,53 @@
...
+            // instantiate new FontFaceObserver for the varient
+            observer = new FontFaceObserver(fontObj.name, fontObj.varient[i]);
+            // add the promise from the observer to the array
+                // 1st argument is text to test font (not needed)
+                // 2nd argument is amount of time to wait for font to load
```
Comment (schalkneethling) [inline kuma/static/js/fonts.js]: Nit: Indentation no consistent for previous two lines.

File: kuma/static/js/fonts.js [hunk 5]
```diff
@@ -1,122 +1,53 @@
...
-        // if all fonts are loaded swap font in
-        if(allLoaded) {
-            ffoFinished();
-
-            // set cookie so fonts are displayed automatically with the server's help
-            var expires = new Date();
-            expires.setDate(expires.getDate() + 1);
-            document.cookie = 'ffo=true; expires=' + expires + '; path=/';
+    // check each font - if not loaded, load using routing above
+    for (var i = 0; i < win.mdn.fonts.length; i++) {
```
Comment (schalkneethling) [inline kuma/static/js/fonts.js]: Cache length of fonts array?

**Comment by jpetto** [comment]
@schalkneethling - Addressed your comments. Thanks for the quick review!

**Pull request merged**